packages/calcite-components/src/components/panel/panel.scss

```diff
@@ -181,6 +166,17 @@
   }
 }
 
+#close,
+#collapse,
+.back-button,
+calcite-action-menu {
+  --calcite-action-background-color: var(--calcite-panel-header-action-background-color);
+  --calcite-action-background-color-hover: var(--calcite-panel-header-action-background-color-hover);
+  --calcite-action-background-color-press: var(--calcite-panel-header-action-background-color-press);
+  --calcite-action-text-color: var(--calcite-panel-header-action-text-color);
+  --calcite-action-text-color-pressed: var(--calcite-panel-header-action-text-color-pressed);
+}
```

Comment:
Does `--calcite-action-corner-radius` need to be set here? Possibly by `  --calcite-internal-panel-corner-radius`?

Reply:
No this is covered by the corner radius of the action containers.